tests/test_misc.py

```diff
@@ -2110,56 +2095,6 @@ def test_large_spends(revault_network, bitcoind, executor):
     )
 
 
-# Tests that getspendtx returns an error when trying to build a spend too big
-# (it wouldn't be possible to announce it to the coordinator when fully signed)
-@pytest.mark.skipif(not POSTGRES_IS_SETUP, reason="Needs Postgres for servers db")
-def test_not_announceable_spend(revault_network, bitcoind, executor):
-    CSV = 2016  # 2 weeks :tm:
-    revault_network.deploy(17, 8, csv=CSV)
-    man = revault_network.man(0)
-
-    vaults = []
-    deposits = []
-    deriv_indexes = []
-    total_amount = 0
-    for i in range(10):
-        amount = random.randint(5, 5000) / 100
-        vaults.append(revault_network.fund(amount))
-        deposits.append(f"{vaults[i]['txid']}:{vaults[i]['vout']}")
-        deriv_indexes.append(vaults[i]["derivation_index"])
-        total_amount += vaults[i]["amount"]
-    revault_network.activate_fresh_vaults(vaults)
-
-    feerate = 1
-    n_outputs = 4
-    fees = revault_network.compute_spendtx_fees(feerate, len(deposits), n_outputs)
-    destinations = {
-        bitcoind.rpc.getnewaddress(): (total_amount - fees) // n_outputs
-        for _ in range(n_outputs)
-    }
-
-    # Hey, this spend is huge!
-    with pytest.raises(
-        RpcError, match="Spend transaction is too large, try spending less outpoints'"
-    ):
-        man.rpc.getspendtx(deposits, destinations, feerate)
-
-    # One less output is ok though
-    n_outputs -= 1
-    destinations = {
-        bitcoind.rpc.getnewaddress(): (total_amount - fees) // n_outputs
-        for _ in range(n_outputs)
-    }
-    spend_tx = man.rpc.getspendtx(deposits, destinations, feerate)["spend_tx"]
-    for man in revault_network.mans():
-        spend_tx = man.man_keychain.sign_spend_psbt(spend_tx, deriv_indexes)
-        man.rpc.updatespendtx(spend_tx)
-    spend_psbt = serializations.PSBT()
-    spend_psbt.deserialize(spend_tx)
```

Comment:
It still make sense to have. Why removing it?

Reply:
So you state:

> Limits are really high since the base64 update! We can't test with
    a spend that is not announceable (the limits are ~1000 inputs,
    ~1000 outputs, ~30 stakes, ~20 mans: just activating the vaults
    goes in timeout!). test_large_spends is checking that we can do big
    spends anyways.
 2 files changed, 1 insertion(+), 54 deletions(-)

That does not sound right. We have a *4 gain, not an infinite one?

Reply:
- It took literally hours to figure out good parameters to use
- Once I figured them out, ooooopsie there are too many inputs the activate_vaults goes in timeout!
- Daniela cries and deletes everything
- What was I testing, anyways?
That big spends can be done? We test that in test_large_spends
That small spends can be done? We test that *somewhere*

Reply:
> What was I testing, anyways?

That a too big spend can't be created instead of failing at announcement time?

Reply:
You're right, I was wrong, I didn't have that many inputs! Checking again...

Reply:
Now I get my mistake: I would stupidly lower the amount of inputs trying to make the test faster, even tho this caused even more outputs to be added, leading to unrealistic numbers (1000 and more). Now hopefully it's fine